src/component/checkboxes.html

```diff
@@ -0,0 +1,16 @@
+<template>
+
+  <template containerless repeat.for="option of options">
+    <div class="checkbox">
```

Comment:
As these styles are bootstrap specific, I made sure to follow bootstrap's documentation. This is how bootstrap says I should do it.